media-libs/libcamera/libcamera-0.4.0.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 2021-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+DESCRIPTION="A complex camera support library for Linux, Android, and ChromeOS"
+HOMEPAGE="https://libcamera.org/"
+
+if [[ ${PV} == "9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.libcamera.org/libcamera/libcamera.git"
+else
+	SRC_URI="https://github.com/kbingham/libcamera/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Please use https://gitlab.freedesktop.org/camera/libcamera/-/releases/v0.4.0 instead. We lobbied upstream hard to add that.